mws/resource_mws_customer_managed_keys.go

```diff
@@ -13,7 +13,7 @@ import (
 // AwsKeyInfo has information about the KMS key for BYOK
 type AwsKeyInfo struct {
 	KeyArn    string `json:"key_arn"`
-	KeyAlias  string `json:"key_alias"`
+	KeyAlias  string `json:"key_alias" tf:"optional"`
```

Comment:
It should only be sent if it is set rather than all the time.

Reply:
yes, then use `omitempty` instead of `tf:optional`. The former won't send data and will mark this attribute as optional in the Terraform, while the latter will only mark it as optional in Terraform and empty value (empty string, 0, false) will be always sent to REST API.